packages/@aws-cdk/aws-appsync/README.md

```diff
@@ -334,6 +334,29 @@ new route53.CnameRecord(this, `CnameApiRecord`, {
 });
 ```
 
+## Log Group
+
+AppSync automatically create a log group with the name `/aws/appsync/apis/<graphql_api_id>` upon deployment with
+log data set to never expire. If you want to set a different expiration period, use the `logConfig.retention` property.
+
+To obtain the GraphQL API's log group as a `logs.ILogGroup` use the `logGroup` property of the
+`GraphqlApi` construct.
+
+```ts
+import * as logs from '@aws-cdk/aws-logs';
+
+const logConfig: LogConfig = {
+  retention: RetentionDays.ONE_WEEK,
+};
+
+new GraphqlApi(this, 'GraphqlApi', {
+  authorizationConfig: {},
+  name: 'myApi',
+  schema: Schema.fromAsset(join(__dirname, 'myApi.graphql')),
+  logConfig,
+});
```

Comment:
These imports need to be rendered by Rosetta in a way that would create a working code snippet. For instance, `new GraphqlApi` would need to be `new appsync.GraphqlApi`.

Reply:
Ok, that's good to know :)